src/olympia/addons/serializers.py

```diff
@@ -114,6 +129,13 @@ class Meta:
                   'is_strict_compatibility_enabled', 'license', 'reviewed',
                   'url', 'version')
 
+    def to_representation(self, instance):
+        # Help the LicenseSerializer find the version we're currently
+        # serializing.
+        if 'license' in self.fields and instance.license:
```

Comment:
`instance` is a `Version`, so it should be a FK and `instance.license` should always work